test/terra/backends/qasm_simulator/qasm_cliffords.py

```diff
@@ -255,7 +255,53 @@ def test_pauli_gate_deterministic_default_basis_gates(self):
                       self.SIMULATOR,
                       shots=shots,
                       basis_gates=basis_gates,
-                      **self.BACKEND_OPTS)
+                      **self.BACKEND_OPTS, optimization_level=0)
+        result = job.result()
+        self.assertSuccess(result)
+        self.compare_counts(result, circuits, targets, delta=0)
+
+    # ---------------------------------------------------------------------
+    # Test identity gate
+    # ---------------------------------------------------------------------
+    def test_id_gate_deterministic(self):
```

Comment:
Arent these tests getting removed in #1068 anyway?

Reply:
Not for the stabilizer simulator